basics/android_app/app_src/build.gradle

```diff
@@ -0,0 +1,10 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+plugins {
+    id 'com.android.application' version '7.2.0' apply false
+    id 'com.android.library' version '7.2.0' apply false
```

Comment:
do you need this? we don't have a library module in the ramses-composer repo, right?